vector/src/main/java/im/vector/app/features/onboarding/ftueauth/SplashCarouselItem.kt

```diff
@@ -35,7 +35,7 @@ abstract class SplashCarouselItem : VectorEpoxyModel<SplashCarouselItem.Holder>(
 
         holder.view.setBackgroundResource(item.pageBackground)
         holder.image.setImageResource(item.image)
-        holder.title.setText(item.title)
+        holder.title.text = item.title.charSequence
```

Comment:
hopefully there is not Emoji there.

Reply:
:sweat_smile:   we should be okay as we've using the `EpoxyCharSequence` and there's no `setTextFuture` :crossed_fingers: